### ISSUE TITLE
NPE script needs to be fixed

### ISSUE THREAD

**Issue by b-remy** [Ciela-Institute/CosmOrford#26]
- [ ] Add wandb losses tracking
- [ ] Fix NPE training script

**Comment by Justinezgh** [comment]
To check/do: 

- [ ] add loss function on wandb
- [ ] check if nb of simulations is the pb
- [ ] test nf architecture from sbi package
- [ ] test with 2d summary statistics instead of 8
- [ ] test FoM from the compressor directly
